_posts/2022-06-29-omero-5-6-5.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: Release of OMERO.server 5.6.5
+---
+
+Today we are releasing OMERO.server 5.6.5 which includes:
+
+-   a new server configuration allowing to control the pyramidal requirement for floating-point images
+-   an upgrade of Bio-Formats to the last released version 6.10.0, see the [release announcement]({{ site.baseurl }}/2022/05/31/bio-formats-6-10-0.html).
+-   the [OMEZarrReader](https://github.com/ome/ZarrReader/) version 0.3.0
+
+The OMEZarrReader inclusion allows the server to import imaging data written according to the 
+[OME-NGFF](https://ngff.openmicroscopy.org/) specification. The reader supports OME-NGFF datasets
+for all published versions of the specification, currently up to 0.4 and reads OME-XML
+metadata stored using to the layout written by Glencoe's
+[bioformats2raw](https://github.com/glencoesoftware/bioformats2raw) utility and currently under
```

Comment:
```suggestion
[bioformats2raw](https://github.com/glencoesoftware/bioformats2raw) utility. The OMEZarrReader is currently under
```

Reply:
I am afraid you lost me there completely. I would suggest you reformulate the very long sentence yourself please, always stating the subject of the action in the sentence anew (not using ``it`` too much).  It seems that the complexity of the problem asks for at least one additional sentence.